evm_arithmetization/src/witness/memory.rs

```diff
@@ -204,7 +217,29 @@ impl MemoryState {
             segment,
             segment.bit_range()
         );
-        val
+        Some(val)
+    }
+
+    pub(crate) fn get(
+        &self,
+        address: MemoryAddress,
+        is_interpreter: bool,
```

Comment:
Unneeded?

Reply:
I think so. And I have the impression that consequently many other methods using this flag also don't need it.

Reply:
It is now used: I had forgotten to include it in the if (thus having unnecessary checks)